config/rbac/bindings/component_role_binding.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRoleBinding
+metadata:
+  name: component-rolebinding
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: pod-viewer-role
+subjects:
+- kind: ServiceAccount
+  name: default
+  namespace: default
```

Comment:
The pod-viewer is a ClusterRole, I am assuming this is for maintenance purpose, user given this role would be able to see all pods across the namespaces.

Reply:
Yes, all our roles are at the cluster scope for the moment.